app/components/Comments/CommentDisplay.jsx

```diff
@@ -173,7 +179,7 @@ export class CommentDisplay extends React.PureComponent {
 
   actionReply() {
     if (!this.ensureAuthenticated())
-      return
+      return null
```

Comment:
not quite fan but this is not core of this PR so nevermind

Reply:
Actually this is the practice recommended by React : https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
Returning `null` is still better than returning `undefined`